main.py

```diff
@@ -36,6 +26,7 @@
 api_id = int(os.getenv('TELEGRAM_API_ID', 0))
 api_hash = str(os.getenv('TELEGRAM_API_HASH'))
 chat_send_to = str(os.getenv('TELEGRAM_CHAT_SEND_TO'))
+messages_collecting_is_on = str(os.getenv('MESSAHES_COLLECTING_IS_ON'))

```

Comment:
spelling mistake

Reply:
you need also to add MESSAGES_COLLECTING_IS_ON to the env-example file

Reply:
done I guess